index.js

```diff
@@ -11,6 +37,25 @@ module.exports = {
     };
   },
 
+  included: function(app) {
+    app.options.autoRun = false;
+    app.options.storeConfigInMeta = false;
+  },
```

Comment:
@rwjblue turns out removing this is not as easy as I thought – contrary to the [documentation](https://github.com/ember-cli/ember-cli/blob/master/ADDON_HOOKS.md#config) (see "Uses"), these options can't actually be set through the `config` hook :( (It makes sense, mutating these *build options* at a random time during build probably has the same problem as setting `autoboot` during boot :wink:)

@stefanpenner can you c/d? If that's true I can (eventually) send a doc PR